src/mito2/src/test_util.rs

```diff
@@ -168,8 +182,14 @@ impl TestEnv {
         let mut builder = Fs::default();
         builder.root(&data_path);
         let object_store = ObjectStore::new(builder).unwrap().finish();
-
-        (log_store, object_store)
+        let object_store_manager = ObjectStoreManager::try_new(
+            vec![("default".to_string(), object_store)]
+                .into_iter()
+                .collect(),
+            "default",
+        )
+        .unwrap();
```

Comment:
Added this method with `testing` feature because I think that we shouldn't use it except test code.

[2fb197f](https://github.com/GreptimeTeam/greptimedb/pull/2643/commits/2fb197f679a44589b908bf15a00b0efec483df5f)

Reply:
As an independent crate, I think it's fine for the `ObjectStoreManager` to provide a method that only constructs with a default object store.

Reply:
> As an independent crate, I think it's fine for the ObjectStoreManager to provide a method that only constructs with a default object store.

Addressed it. [dbd0603](https://github.com/GreptimeTeam/greptimedb/pull/2643/commits/dbd0603421f7192251ad2fee7305c70440a0e49d)